pkg/vetter/util/util_test.go

```diff
@@ -122,3 +122,87 @@ var _ = Describe("Converting configmap to SidecarInjectionSpec", func() {
 	})
 
 })
+
+var _ = Describe("Test ProxyStatusPort", func() {
+	It("Finds an override that is not 15020", func() {
+
+		// Actual list from a kubectl get pods ratings-v1-76f4c9765f-n8w45 -o yaml call (so what we should have
+		//  at run time.
+		testArgs := []string{"proxy",
+			"sidecar",
+			"--domain",
+			"$(POD_NAMESPACE).svc.cluster.local",
+			"--configPath",
+			"/etc/istio/proxy",
+			"--binaryPath",
+			"/usr/local/bin/envoy",
+			"--serviceCluster",
+			"atings.$(POD_NAMESPACE)",
+			"--drainDuration",
+			"5s",
+			"--parentShutdownDuration",
+			"m0s",
+			"--discoveryAddress",
+			"istio-pilot.istio-system:15011",
+			"--zipkinAddress",
+			"zipkin.istio-system:9411",
+			"--connectTimeout",
+			"0s",
+			"--proxyAdminPort",
+			"15000",
+			"--concurrency",
+			"2",
+			"--controlPlaneAuthPolicy",
+			"UTUAL_TLS",
+			"--statusPort",
+			"15020",
+			"--applicationPorts",
+			"9080",
+		}
+		container := corev1.Container{Args: testArgs}
```

Comment:
It is ok for now but I would suggest using a table driven approach here so simplify the test.